docs/data/material/varnish/colors/LightCategoricalColorValues.js

```diff
@@ -7,7 +7,7 @@ import { lightCategoricalColor, dictionaryToArray } from '@allenai/varnish';
 import { ChartingGrid, ColorValues } from './ColorValues';
 
 const lightCategoricalColorGroup = {
-  darkCategorical: dictionaryToArray(lightCategoricalColor),
+  lightCategorical: dictionaryToArray(lightCategoricalColor),
```

Comment:
learning question: why are we making the change here from dark to light categorical?

Reply:
this was a copy paste error when i added the colors earlier...
this should always have been light